ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/SimpleIdentityManager.java

```diff
@@ -0,0 +1,122 @@
+package com.iotics.sdk.identity;
+
+
+import com.iotics.sdk.identity.jna.JnaSdkApiInitialiser;
+import com.iotics.sdk.identity.jna.SdkApi;
+
+import java.time.Duration;
+
+/**
+ * High level class to manage a user and an agent identities plus wrapper methods to create token and twin
+ */
+public class SimpleIdentityManager implements IdentityManager {
+
+    private final Identity agentIdentity;
+    private final Identity userIdentity;
+    private final SimpleIdentity idSdk;
+
+    private SimpleIdentityManager(String resolverAddress,
+                                  String userSeed, String agentSeed,
+                                  String userKeyName, String userKeyID,
+                                  String agentKeyName, String agentKeyID,
+                                  String authDelegationID) {
+        SdkApi api = new JnaSdkApiInitialiser().get();
```

Comment:
Is not better for things to be explicitly created via a method after construction instead of in the constructor?